roles/openshift_hosted/templates/registry_config.j2

```diff
@@ -63,7 +63,7 @@ storage:
     keyfile: /etc/registry/gcs.json
 {%   endif -%}
 {%   if openshift_hosted_registry_storage_gcs_rootdirectory is defined %}
-    rootdirectory: {{ openshift_hosted_registry_storage_gcs_rootdirectory }}
+    rootdirectory: {{ openshift_hosted_registry_storage_gcs_rootdirectory | default('/registry') }}
```

Comment:
This change seems unrelated.

Reply:
Yeah, this is unrelated to swift issue, but it should another minor issue, if need open another PR to track gcs default rootdirectory issue, I will open another PR for that.

Reply:
no, this is fine